ELE-3702 Adding documentation for new Annotation events (#30)
talis/advanced-mis: topics/event-classes.md

| `player.view.contributors` | A resource was opened in the player by a contributor (academic, teaching assistant) |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-view-contributors)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
| `player.annotate.createannotation` | A new annotation (comment thread) was created by a user |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-annotate-createannotation)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
| `player.annotate.createannotation.createcomment` | A new class annotation (comment thread) was created by a user |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-annotate-createannotation)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |

> | `player.annotate.createannotation.personalnote` | A new personal annotation (comment thread) was created by a user |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-annotate-createannotation)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |

Reviewer: A minor thing but personal notes are not threaded and you can't reply to existing class comment threads with them either.

```suggestion
| `player.annotate.createannotation.personalnote` | A new personal annotation (private note) was created by a user |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-annotate-createannotation)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
```

| `player.annotate.createannotation.anonymouscomment` | A new anonymous annotation (comment thread) was created by a user |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-annotate-createannotation)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |

> | `player.annotate.createreply` | A new replay (to an existing thread) was created by a user |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-annotate-createreply)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |

Reviewer: Whilst we're here I spotted this typo:

```suggestion
| `player.annotate.createreply` | A new reply (to an existing thread) was created by a user |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-annotate-createreply)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
```

| `player.view` | A resource was opened in the player by a student. `digitisation.view` is also recorded by the Talis Elevate player when the content has originated from Copyright Clearance. This is in addition to `player.view`. |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-view)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
| `player.view.contributors` | A resource was opened in the player by a contributor (academic, teaching assistant) |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-view-contributors)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
| `player.annotate.createannotation` | A new annotation (comment thread) was created by a user |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-annotate-createannotation)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |

> | `player.annotate.createannotation.createcomment` | A new annotation (comment thread) was created by a user |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-annotate-createannotation-createcomment)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |

Reviewer 2: ```suggestion
| `player.annotate.createannotation.createcomment` | A new class annotation (comment thread) was created by a user |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-annotate-createannotation-createcomment)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
```

> | `player.annotate.createannotation.personalnote` | A new annotation (comment thread) was created by a user |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-annotate-createannotation-personalnote)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |

Reviewer 2: ```suggestion
| `player.annotate.createannotation.personalnote` | A new personal annotation (comment thread) was created by a user |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-annotate-createannotation-personalnote)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
```

> | `player.annotate.createannotation.anonymouscomment` | A new annotation (comment thread) was created by a user |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-annotate-createannotation-anonymouscomment)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |

Reviewer 2: ```suggestion
| `player.annotate.createannotation.anonymouscomment` | A new anonymous annotation (comment thread) was created by a user |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-annotate-createannotation-anonymouscomment)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
```
